src/kirby/components/icon/icon.component.tns.global.scss

```
@@ -1,15 +1,26 @@
 @import '../../scss/native-imports';
  
-Label {
-    font-family: "Ionicons", "ionicons";
+.sm {
```

Ensure you only target the icon when moving styles to global:
```suggestion
kirby-icon.sm {
```

---

As i wrote in the [description](https://github.com/kirbydesign/designsystem/pull/286#issue-280359718) of this PR, i can not make it work, if i use the component name as selector on native. 

That is why i added the `kirby-icon` css class to the `Label` on native.